app/models/post.rb

```diff
@@ -0,0 +1,6 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments, dependent: :destroy
+  validates :title, presence: true
+  validates :body, presence: true
```

Comment:
You could also write this as `validates :title, :body, presence: true`

Reply:
Ok, thanks for this tip.